omega/omega-transport/omega-transport-dubbo/src/test/java/SagaDubboProviderFilterTest.java

```diff
@@ -45,7 +45,6 @@
   @Before
   public void setUp() {
     omegaContext.clear();
-    ApplicationContext applicationContext = mock(ApplicationContext.class);
```

Comment:
JUnit always creates a new mock applicationcontext instance for the member field.  But this applicationContext is a different scope.  There is some miss part need to be add.

Reply:
yes, I have fix it at this commit : 2fda9f6, but The Travis CI build failed